app/src/main/res/values/styles.xml

```diff
@@ -339,4 +413,21 @@
     <item name="fontFamily">sans-serif</item>
     <item name="textAllCaps">false</item>
   </style>
+
+  <style name="TextViewStart" parent="TextView.Common1">
+    <item name="android:textAlignment">viewStart</item>
+  </style>
+
+  <style name="TextViewCenter" parent="TextView.Common1">
```

Comment:
Would 'CenterVerticalHorizontal' be clearer to distinguish from the other two center options? Ditto for other similar styles.

Reply:
Should I rename it to `TextViewCenterVerticalHorizontal` or just `CenterVerticalHorizontal`?

Reply:
Also, I think `TextViewCenter` seems like more general as it defines by default both the vertical and horizontal axis.

Reply:
Hmm after taking another pass, I think this is fine for the medium term.